source/initial-dns-seedlist-discovery/tests/README.md

```diff
@@ -3,6 +3,31 @@
 This directory contains platform-independent tests that drivers can use to prove their conformance to the Initial DNS
 Seedlist Discovery spec.
 
+### Prose Tests
+
+1. When given the following valid SRVs, parse the connection string without throwing a parse related error.
+
+   a. Only one domain level (ex: `mongodb+srv://localhost`).
+
+   b. Only two domain levels (ex: `mongodb+srv://mongo.local`).
+
+<br />
+
+2. When given an SRV that does NOT end with the original SRV's `{domainname}`, throw a runtime error.
+
+   For example, the SRV `mongodb+srv://blogs.mongodb.com` resolving to `blogs.evil.com` should prompt an error.
+
+   Run this test for the following test cases: SRVs with one, two, and three `.` separated parts.
```

Comment:
This should probably be the start of the test, "for the following steps run them for each of these cases" followed by what should be checked for each, it implies the table structure we usually employ but more importantly helps the author deduplicate assertions and steps they have to perform

Reply:
Okay sounds good, changed the wording.